charts/kaito/workspace/templates/deployment.yaml

```diff
@@ -47,6 +47,8 @@ spec:
               value: {{ .Values.presetRegistryName }}
             - name: CLOUD_PROVIDER
               value: {{ .Values.cloudProviderName }}
+            - name: CLUSTER_NAME
+              value: {{ .Values.clusterName }}
```

Comment:
consider 
value: {{ .Values.clusterName | default "default-cluster-name" }}

Reply:
added default to values.yaml